site/content/docs/_index.md

```diff
@@ -40,26 +40,27 @@ break or slow down.
 >}}
 </div>
 
-## Diving deeper
+## Adopt The Monitoring As Code Workflow
 
-Checkly offers a lot of power and perks to modern developers and DevOps teams. Manage checks at scale with Terraform, hook into your CI/CD and deployment flow.
+Checkly offers a lot of power and perks to modern developers and DevOps teams to manage their Checks and monitoring setup from their code base. We call it "monitoring as code".
```

Comment:
@tnolet nit: Monitoring as Code? (If we call it MaC and Infrastructure as Code is IaC).

Reply:
new writing as per Jeff is lower cased